### ISSUE TITLE
Publishing Results to Registrant Results page should not depend on 'competition awarded?'

### ISSUE THREAD

**Issue by scotthue** [rdunlop/unicycling-registration#558]
Currently the behavior is not predictable. Maybe Registrant Results should just be available as soon as the competition is publish? Or there needs to be a more logical behavior at least.

(https://register.ecunicycling2017.eu/en/registrants/1/results)

**Comment by rdunlop** [comment]
The results here are based on "awarded?" status for each competition. Are you saying that even with that knowledge, they are "unpredictable"?

**Comment by scotthue** [comment]
No. If one knows that, it's okay I guess. But it's quite hard to know that
without being told.

**Comment by rdunlop** [comment]
I tried to explain it once. I welcome better ideas on how to inform people.

<img width="917" alt="screenshot 2017-08-03 10 17 32" src="https://user-images.githubusercontent.com/1149122/28929159-fd1f1844-7834-11e7-91d6-12c30c8c3661.png">


I'm currently considering removing the "Awarded" status from a competition..making "published" the final status...thus removing this problem. thoughts on that idea?

**Comment by scotthue** [comment]
I don't think there is a lot of worth in the Awarded status.

Having "Published" as the final state is fine with me.
